packages/variable-editor/src/VariableEditor.tsx

```diff
@@ -30,7 +30,8 @@ function VariableEditor(props: EditorProps) {
     return {
       data: () => ['variable-editor', 'data', context],
       saveData: () => ['saveData'],
-      validate: () => ['validate']
+      validate: () => ['validate'],
+      knownVariables: () => ['meta/knownVariables']
```

Comment:
we should add here more info the query key, otherwise, this will maybe clash in the global neo react query context. E.g. like here https://github.com/axonivy/form-editor-client/blob/master/packages/editor/src/components/editor/Editor.tsx#L34-L35